includes/addons.php

```diff
@@ -88,6 +95,108 @@ function pmpro_getAddonBySlug( $slug ) {
 	return false;
 }
 
+/**
+ * Get Add Ons by license type.
+ *
+ * @since 2.8.x
+ *
+ * @param array $license An array of license type for our Add Ons.
+ * @param bool $include_hidden Optionally include hidden Add Ons in the return array.
+ * @return array $addons An array of all plugin objects found or an empty array if none found.
+ */
+function pmpro_get_addons_by_license_types( $license_types = array(), $include_hidden = false ) {
+	// Return if we don't have a license to search.
+	if ( empty( $license_types ) ) {
+		return array();
+	}
+
+	// Get all Add Ons. Return if empty.
+	$addons = pmpro_getAddons();
+	if ( empty( $addons ) ) {
+		return array();
+	}
+
+	// Note: Do we need to make sure that $license_types is an array here?
```

Comment:
I think it'll be good to ensure it's an array, if not cast it to an array to prevent any warnings.

Reply:
I removed this function now actually because it isn't in use. I saved it in a private gist if we need to bring it back.